includes/features/SU-SWS/stanford_private_page/stanford_private_page.feature

```diff
@@ -3,16 +3,6 @@ Feature: Stanford Private Page
   As all types of users
   I want to check for CRUD permissions.
 
-  @api @javascript @dev @destructive
-  Scenario: Create Private Page
-    Given I am logged in as a user with the "site owner" role
-    When I wait for the Site Actions drop down to appear
-    And I click "Add Private Page" in the "Admin Shortcuts" region
-    Then I should see "Create Private Page" in the "Branding" region
-    Then I fill in "edit-title" with "Private Page Test"
-    And I press the "Save" button
-    Then I should see "Private Page Private Page Test has been created" in the "Console" region
-
   @api @dev @destructive
   Scenario: Site owner can create and edit Private Page types
     Given I am logged in as a user with the "site owner" role
```

Comment:
We should use the administrator role when testing creation of content types in the shared directory.